include/deal.II/lac/constraint_matrix.templates.h

```diff
@@ -639,15 +651,15 @@ distribute_local_to_global (const Vector<LocalType>      &local_vector,
         // constrained. if not, just write the entry
         // into the vector. otherwise, need to resolve
         // the constraint
-        if (is_constrained(local_dof_indices[i]) == false)
+        if (is_constrained(local_dof_indices_col[i]) == false)
           {
-            global_vector(local_dof_indices[i]) += local_vector(i);
+            global_vector(local_dof_indices_col[i]) += local_vector(i);
```

Comment:
From here (line 642(old)/654(new)) we see that global_vector is indexed with variable i, afterwards we index the local matrix by `local_matrix(j,i)`, so i is the column index and j the row index.
Hence the global vector is indexed by `local_dof_indices_col` in this function. Sry that I cannot exactly explain why, but at least I hope it became clear that it is consistent with the previous code...